framework/src/userobject/ElementSubdomainModifier.C

```diff
@@ -0,0 +1,353 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "ElementSubdomainModifier.h"
+#include "DisplacedProblem.h"
+
+#include "libmesh/parallel_algebra.h"
+#include "libmesh/parallel.h"
+#include "libmesh/dof_map.h"
+
+#include "libmesh/parallel_ghost_sync.h"
+#include "libmesh/mesh_communication.h"
```

Comment:
We may not need this header if it is already a parallel object

Reply:
Removed.